localization/ndt_scan_matcher/README.md

```diff
@@ -255,3 +255,79 @@ initial_pose_offset_model is rotated around (x,y) = (0,0) in the direction of th
 initial_pose_offset_model_x & initial_pose_offset_model_y must have the same number of elements.
 
 {{ json_to_markdown("localization/ndt_scan_matcher/schema/sub/covariance_covariance_estimation.json") }}
+
+## Diagnostics
+
+### sensor_points_callback
+
+<img src="./media/diagnostic_sensor_points_callback.png" alt="drawing" width="600"/>
+
+| Name                                      | Description                                                                            | Transition condition to OK                                                                                                                                                                                          | Transition condition to Warning                                                                                                                                                                                                                                                                                                                                          | Transition condition to Error            | Whether to reject the estimation result (affects `skipping_publish_num`) |
```

Comment:
"Transition condition to OK" is a bit confusing because the default condition of diagnostics is OK. 
I think this column may be delete.